pkg/webhooks/pytorch/pytorchjob_webhook.go

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright 2024 The Kubeflow Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pytorch
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"golang.org/x/exp/slices"
```

Comment:
After we upgrade the Go version to v1.22, we can get rid of this dependency since the "slices" library has graduated to the standard library in Go 1.22. But I can not submit PR since this #2043 is blocking my bumping v1.22 PR 😓 I'm on the whence whether or no I should create PR and then I should close #2043.

WDYT?

Reply:
Since #2043 is merged into the master branch, I will upgrade the Go version in another PR.

Reply:
Submitted PR: #2046

Reply:
update now?

Reply:
Yes, we can update this.

Reply:
Done.